topics/yaml/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: yml
+display_name: YAML
+github_url: https://github.com/yaml/
+logo: yaml.png
+released: May 11, 2001
+short_description: YAML is a human friendly data serialization standard.
+topic: yaml
+url: https://yaml.org/
+wikipedia_url: https://en.wikipedia.org/wiki/YAML
+---
+YAML is a human friendly data serialization standard for all programming languages. It is commonly used for configurations.
```

Comment:
I think it'd be valuable to add what `YAML` stands for in this description:
> YAML: YAML Ain't Markup Language

What do you think?

Reply:
That is great. Didn't know how I missed that. Added the abbreviation.